server/controllers/sampleCtrl.js

```diff
@@ -75,6 +76,8 @@ async function getIndex(req, res, next) {
       { label: 'Node', url: serverConfig.hostUrl },
     ]
 
+    const logotypeSrc = `${proxyPrefix}/assets/logotype/${theme === 'external' ? 'logotype-white.svg' : 'logotype-blue.svg'}`
```

Comment:
Since this is only used by kthHeader.handlebars, do you think we can move it there? In that case, this file wouldn't need to be touched (i.e. the theme could be defined in the same way as before)

Reply:
Yep. Couldn’t agree more. Maybe a handlebar helper? Could be implemented in this app first, and then moved to `@kth/kth-node-web-common`.